src/containers/ConnectorManagement.jsx

```diff
@@ -106,6 +109,14 @@ export default class ConnectorManagement extends Component {
     this.gotoParent()
   }
 
+  handleTimeout () {
+    this.setState({ jobInBackground: true })
+  }
+
+  accountConfig () {
+    this.setState({ jobInBackground: false })
```

Comment:
Does it work ?

Actually the role of this method is to display the edition modal with the newly created account. So to be in adequation with the logic here https://github.com/cozy/cozy-collect/blob/master/src/containers/ConnectorManagement.jsx#L87, this method should set the state property `isConnected` to true instead.